collections/src/main/scala/strawman/collection/LazyZipOps.scala

```diff
@@ -0,0 +1,352 @@
+package strawman.collection
+
+import scala.{Boolean, StringContext, Unit}
+import scala.language.implicitConversions
+
+final class LazyZipOps[A, C1 <: Iterable[A]] private[collection](`this`: C1) {
```

Comment:
We could even make it a value class (even though I bet the saving wouldn’t be significant at all)

Reply:
this is a very easy change, so I think it's worth doing it despite how much saving we eventually get.